app/views/shared/_consumer_home_address_fields.html.erb

```diff
@@ -102,13 +102,13 @@
         <% end %>
       </div>
     <% end %>
+
+    <% if f.object.addresses[1].present? %>
+      <span tabindex="0" onkeydown="handleButtonKeyDown(event, 'remove_mailing_address')" class="mb-4 form-action btn-secondary <%= pundit_class Family, :updateable?%>" id="remove_mailing_address" data-cuke="remove_mailing_address">Remove Mailing Address</span>
+    <% else %>
+      <span tabindex="0" onkeydown="handleButtonKeyDown(event, 'add_mailing_address')" id="add_mailing_address" class="mb-4 form-action btn-secondary <%= pundit_class Family, :updateable?%>">Add Mailing Address</span>
```

Comment:
this has been reworked as pr currently in review, verify these changes against that